packs/cargo/cabins.pack.json

```diff
@@ -591,7 +591,7 @@
         },
         "maps/cargo-ship/cabins2.json/entities/141/settings/event/39/message": {
                 "orig": "That is the other thing...\\. there really should not be any Evotars in CrossWorlds at all.",
-                "text": "C'est ça l'autre truc...\\. Il ne devrait pas y avoir d'Évotars dans CrossWorlds."
+                "text": "C'est cela et aussi...\\. il ne devrait pas y avoir d'Évotars dans CrossWorlds.",
```

Comment:
Did you edit those files by hand ? because `make generate` fails quickly.

Reply:
Yes